### PR TITLE
Add relative impact overall to LSR

### LINKED ISSUE #1787: Add % earnings impact to overall LSR chart
We currently show the % change to earnings in the decile charts but not overall

### LINKED ISSUE #1842: Re-enable relative decile LSR impacts
Pursuant to pushing of https://github.com/PolicyEngine/policyengine-api/pull/1547

### PR DESCRIPTION
**Pull request by anth-volk**
## Description

Fixes #1787.
Fixes #1842.

## Changes

This PR adds an overall impact component to the labor supply response output section of the policy calculator. It also re-enables the decile relative LSR impact pages since pushing https://github.com/PolicyEngine/policyengine-api/pull/1547.

## Screenshots

https://github.com/PolicyEngine/policyengine-app/assets/14987227/e602649e-4fd8-4f5c-8712-6027ed3153a1

## Tests

None have been added.

### CODE REVIEW

File: src/pages/policy/output/labourSupply/LaborSupplyResponseAbsolute.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+              : ["total"],
+          textposition: "inside",
+          text: values.map((value) =>
+            formatCurrencyAbbr(value * 1e9, metadata.countryId, {
+              maximumFractionDigits: 1,
+            }),
+          ),
+          increasing: { marker: { color: style.colors.BLUE } },
+          decreasing: { marker: { color: style.colors.DARK_GRAY } },
+          // Total should be dark gray if negative, dark green if positive
```
Comment (MaxGhenis) [inline src/pages/policy/output/labourSupply/LaborSupplyResponseAbsolute.jsx:37]:
```suggestion
          // Total should be dark gray if negative, dark blue if positive
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
The total relative earnings response will differ from the average of the deciles' relative earnings responses. You could calculate it as a weighted average by earnings, but I'd suggest we instead emit it directly from the API.

File: src/pages/policy/output/ImpactTypes.jsx [hunk 2]
```diff
@@ -13,26 +13,25 @@ import povertyImpactByGender from "./poverty/PovertyImpactByGender";
...
   LabourSupplyDecileRelativeImpactTotal,
 } from "./labourSupply/LabourSupplyDecileRelativeImpacts";
-*/
 import Analysis from "./Analysis";
 
 const map = {
   "budgetaryImpact.overall": budgetaryImpact,
   "budgetaryImpact.byProgram": detailedBudgetaryImpact,
-  "laborSupplyImpact.earnings.overall": laborSupplyImpact,
+  // "laborSupplyImpact.earnings.overall": laborSupplyImpact,
```
Comment (MaxGhenis) [inline src/pages/policy/output/ImpactTypes.jsx:34]: are we waiting on an api change to uncomment this?

Comment (anth-volk) [reply src/pages/policy/output/ImpactTypes.jsx:34]: Ah, no, that's just garbage, thanks for finding

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**